File: app_python/src/timer/service.py
import datetime
import pytz


class Timer:
    """

    Timer is a class that provides simple interface to work with time

    """

    def __init__(self):
        self.MoscowTimezone = pytz.timezone("Europe/Moscow")

    def now(self):
        return datetime.datetime.now(tz=self.MoscowTimezone)

File: app_go/internal/pkg/service/timer/service.go
package timer

import (
	"context"
	"fmt"
	"time"

	"wiirtex/devops_labs/internal/pkg/service/timer/types"
)

var _ types.Timer = Timer{}

type Timer struct {
	MoscowTimezone *time.Location
}

func New() Timer {
	loc, err := time.LoadLocation("Europe/Moscow")
	if err != nil {
		panic(fmt.Errorf("can not load location: %w", err))
	}
	return Timer{
		MoscowTimezone: loc,
	}
}

// Now implements types.Timer interface
func (t Timer) Now(_ context.Context) time.Time {
	return time.Now().In(t.MoscowTimezone)
}
